promgen/templates/promgen/host_form.html

```diff
@@ -17,7 +17,7 @@ <h1>Farm: {{ farm.name }}</h1>
   <table>
     {{ form.as_table }}
   </table>
-    <input type="submit" value="Register Hosts" />
+  <input type="submit" value="{% trans "Register Hosts" %}" />
```

Comment:
Very minor nit, but in this case when I'm mixing some of the templates, I'll often use single quotes in the inside bit, to avoid errors with auto format.

```suggestion
  <input type="submit" value="{% trans 'Register Hosts' %}" />
```

I've had this error before with nested quotes

Reply:
Sure, no problem 👍

Reply:
Fixed in 073e944.